src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs

```diff
@@ -142,24 +140,33 @@ public async Task RecordGatewayRemovedOutcome(Guid applicationId, string userId,
                 if (registerStatus?.OrganisationId != null && (appealStatus == AppealStatus.SuccessfulAlreadyActive ||
                                                                appealStatus == AppealStatus.SuccessfulFitnessForFunding))
                 {
-                    var updateDeterminedDateRequest = new UpdateOrganisationApplicationDeterminedDateRequest
+                    var updateOrganisationRequest = new UpdateOrganisationRequest
                     {
                         ApplicationDeterminedDate = DateTime.UtcNow.Date,
                         LegalName = registrationDetails.LegalName,
                         OrganisationId = registerStatus.OrganisationId.Value,
-                        UpdatedBy = userId
+                        Username = userName,
+                        CharityNumber = registrationDetails.CharityNumber,
+                        CompanyNumber = registrationDetails.CompanyNumber,
+                        OrganisationTypeId = registrationDetails.OrganisationTypeId,
+                        ProviderTypeId = registrationDetails.ProviderTypeId,
+                        TradingName = registrationDetails.TradingName,
                     };
 
-                    await _registerApiClient.UpdateApplicationDeterminedDate(updateDeterminedDateRequest);
```

Comment:
Is the method UpdateApplicationDeterminedDate redundant now? Should we delete it?

Reply:
agree

Reply:
I've raised a task to clean this up APR-2832, as it's already reached PO Review